inference/core/version.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.9.23"
+__version__ = "0.9.23rc5"
```

Comment:
Do you think we should update minor version to 10? Maybe even major version? Seems like significant changes

Reply:
good idea - I will bump at the end

Reply:
I like the idea of going to 1.0 soon. We def are using this extensively in production and would like others to as well.